modules/dreamview/backend/teleop/teleop.cc

```diff
@@ -40,7 +43,8 @@ const std::string modem0_id = "0";
 const std::string modem1_id = "1";
 const std::string modem2_id = "2";
 
-const unsigned int encoder_count = 2;
+const unsigned int encoder_count = 3;
+const unsigned int write_wait_ms = 50;
```

Comment:
Prefer `static`, and names like `kEncoderCount`.

Reply:
fixed [here](https://github.com/ApolloAuto/apollo/pull/10228/commits/f45083489676b39d68f3d82468216844c30de7d1)